maps/graph_edges/glitched_logic/flo_bubble_berry_tree_early.py

```diff
@@ -8,6 +8,12 @@
     {"from": {"map": "FLO_24", "id": 0}, "to": {"map": "FLO_24", "id": "Tree1_Drop1A"}, "reqs": [["can_shake_trees"], ["Lakilester"]], "mapchange": False},
     #* (SE) Water Level Room Exit Left -> Tree1_Drop1B (BubbleBerry)
     {"from": {"map": "FLO_24", "id": 0}, "to": {"map": "FLO_24", "id": "Tree1_Drop1B"}, "reqs": [["can_shake_trees"], ["Lakilester"]], "mapchange": False},
+
+    # Laki can be used to cross this room regardless of water level
+    #? (SE) Water Level Room Exit Left -> (SE) Water Level Room Exit Right
+    {"from": {"map": "FLO_24", "id": 0}, "to": {"map": "FLO_24", "id": 1}, "reqs": [["Lakilester"]]}, 
+    #? (SE) Water Level Room Exit Right -> (SE) Water Level Room Exit Left
+    {"from": {"map": "FLO_24", "id": 1}, "to": {"map": "FLO_24", "id": 0}, "reqs": [["Lakilester"]]}
 ]
 
 edges_flo_add_bubble_berry_tree_early_ultra_boots = [
```

Comment:
Not sure about this one.
By turning on the laki jump trick for getting to the bubble tree, this seems to intend to also put laki jumping to the left and right exits in logic.
But a side-effect of this change is that this also turns on laki-clipping onto the water after doing the Water Stone trade-in

Reply:
I don't personally see an issue with this implication, but I'm also not sure how else it could be handled other than an additional trick that covers both but not the bubble berry tree.

Reply:
I guess we can just roll with this for now